### ISSUE TITLE
The VS217 image needs cygwin and msys2

### ISSUE THREAD

**Issue by DavidStaheli** [Microsoft/azure-pipelines-image-generation#271]
The VS217 image needs cygwin and msys2

**Comment by vtbassmatt** [comment]
I'm fine with this change. Those are relatively common tools for POSIX interop on Windows. @DavidStaheli is this one the vendor can take?

**Comment by DavidStaheli** [comment]
@vtbassmatt, we haven't reached approval for that yet, but hopefully soon.

**Comment by voltagex** [comment]
Would these also work on the win1803 image?

**Comment by vtbassmatt** [comment]
The 1803 image is for running Windows containers only. I'd like to keep it very tightly focused.

**Comment by strangecargo** [comment]
Any movement on this or an ETA?

**Comment by vtbassmatt** [comment]
No, and I'm sorry for the long lag time here. We're working through some issues with the Windows image (namely, it outgrew the space available because we've added a ton of tools lately). We should have this in Q1, and I'm _hoping_ much sooner.

**Comment by junyzheng** [comment]
Any updates on this one?

**Comment by olegchir** [comment]
Any updates?

**Comment by strangecargo** [comment]
Any news on this?
